guide/english/certifications/javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords/index.md

```diff
@@ -1,25 +1,23 @@
 ---
 title: Compare Scopes of the var and let Keywords
 ---
-![](//discourse-user-assets.s3.amazonaws.com/original/2X/3/3c8584a085a0deaea66b3400e6321eeadab552a2.jpg)
 
 ![:triangular_flag_on_post:](https://forum.freecodecamp.com/images/emoji/emoji_one/triangular_flag_on_post.png?v=3 ":triangular_flag_on_post:") Remember to use <a>**`Read-Search-Ask`**</a> if you get stuck. Try to pair program ![:busts_in_silhouette:](https://forum.freecodecamp.com/images/emoji/emoji_one/busts_in_silhouette.png?v=3 ":busts_in_silhouette:") and write your own code ![:pencil:](https://forum.freecodecamp.com/images/emoji/emoji_one/pencil.png?v=3 ":pencil:")
 
-### Problem Explanation:
+### Problem explanation:
 
-We need to change `var` to `let` in our function scope and add `let` to our block scope.
+_Fix the code so that `i` declared in the if statement is a separate variable than `i` declared in the first line of the function._
 
 ## ![:speech_balloon:](https://forum.freecodecamp.com/images/emoji/emoji_one/speech_balloon.png?v=3 ":speech_balloon:") Hint: 1
 
-*   Find `var` and replace with `let`.
+*  _Be certain not to use the `var` keyword anywhere in your code._
```

Comment:
I disagree. The original text gives away the solution so its not actually a hint.

Reply:
> _Fix the code so that `i` declared in the if statement is a separate variable than `i` declared in the first line of the function._

@AdrianSkar This statement is not really clear. One could argue that since both variables are named `i`, they're the same variable, reused in a different scope. To avoid any confusion, please revert to the original sentence.

Reply:
Hi @Manish-Giri! I changed that text because I felt it "gives away" the solution but I didn't write the replacement, I used the [original exercise instructions](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/compare-scopes-of-the-var-and-let-keywords) (that's why I put it in italics). If you disagree we can just let it as it was, from my _newb_ perspective I like it better as in previous guide pages.

Reply:
I agree with @AdrianSkar on this topic, the original text gives away the solution. What I propose is that we let the first hint be as has been suggested by the OP, but with the following formatting modifications:
```suggestion
Be certain not to use the `var` keyword anywhere in your code since variables declared with var are globally scoped.
```

Then we add a second hint with the text:
```text
Find `var` and replace with `let`.
```

Reply:
That's ok in my opinion. I put them in italics because it's a citation from the original instructions but that looks good too. Should I commit this changes?
Regarding the second hint it's actually the solution so not sure about that one. In any case I'm the inexperienced one so whatever you think it's best/could help someone with this lesson.

Reply:
@AdrianSkar Before committing it, I'd say we get the approval of either of @RandellDawson / @moT01 / @Manish-Giri